x-pack/plugins/ml/public/components/form_label/form_label.js

```diff
@@ -4,34 +4,31 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import './styles/main.less';
 
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
 
-import { uiModules } from 'ui/modules';
-const module = uiModules.get('apps/ml');
+import { JsonTooltip } from '../json_tooltip/json_tooltip';
 
-// directive for creating a form label including a hoverable icon
```

Comment:
Isn't it worth keeping in most of this comment which explains what the component does and just remove the Angular parts?

Reply:
I adapted the text here. I originally copied it to `form_label_directive` but you're right of course, once the angular part is gone, we need the comment here.